### ISSUE TITLE
Neuropal segmentation sometimes gives no neurons

### ISSUE THREAD

**Issue by Charles-Fieseler-Vienna** [Zimmer-lab/wbfm#1012]
> So basically the segmentation didn't produce any neurons, which crashes everything... this might just be a question of dim data or otherwise very different data, but it's not clear why
> 
> Now it shouldn't crash and gives proper warnings on my machine, but obviously there are no neuropal segmentation 

 _Originally posted by @Charles-Fieseler-Vienna in [#1009](https://github.com/Zimmer-lab/wbfm/issues/1009#issuecomment-3617500610)_

**Comment by Charles-Fieseler-Vienna** [comment]
I think this is basically just a mismatch of segmentation training data and the NP data... only real todo is to retrain. Can try some pretrained networks but they were never very good
